src/main/java/it/gov/pagopa/rtd/ms/rtdmsfileregister/adapter/BlobRegisterAdapter.java

```diff
@@ -29,8 +26,13 @@ public class BlobRegisterAdapter {
 
   private static final String EVENT_NOT_OF_INTEREST_MSG = "Event not of interest: ";
 
-  @Autowired
-  FileMetadataService fileMetadataService;
+  private final FileMetadataService fileMetadataService;
+  private final NamingConventionPolicy namingConventionPolicy;
+
+  public BlobRegisterAdapter(FileMetadataService fileMetadataService, NamingConventionPolicy namingConventionPolicy) {
```

Comment:
just a tip: if you want to spare these few constructor lines, you can add the `@RequiredArgsConstructor` at class level and lombok creates a constructor for the `final` fields.

Reply:
Done :)